pkg/utils/httpserverutils.go

```diff
@@ -823,3 +825,42 @@ func GetSpecificIdentifier() (string, error) {
 
 	return hostname, nil
 }
+
+func GetDecodedBody(ctx *fasthttp.RequestCtx) ([]byte, error) {
+	contentEncoding := string(ctx.Request.Header.Peek("Content-Encoding"))
```

Comment:
is this header peek/check content sensitive?
In the request that I made I saw it was sent as `content-encoding: gzip`

Reply:
It should be case-insensitive for HTTP/1.1 (see https://datatracker.ietf.org/doc/html/rfc7230#section-3.2). Also it worked for me with this capitalization when I tested it